### PR TITLE
Jsoup with images

### PR DESCRIPTION
**Pull request by mpwint**

### CODE REVIEW

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <script src="https://cdn.ckeditor.com/ckeditor5/11.2.0/classic/ckeditor.js"></script>
```
Comment (lmtrujillo) [inline src/main/webapp/user-page.html:20]: Nice! Didn't know about it, did you noticed speed changes with this tool?

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 2]
```diff
@@ -76,11 +76,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.basicWithImages());
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.basicWithImages());
```
Comment (lmtrujillo) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:79]: Why did you change the variable name? Just to make it more understandable?

**Review by lmtrujillo — APPROVED** [review]

File: src/main/webapp/user-page.html (hunk 1 above)
Comment (mpwint) [reply src/main/webapp/user-page.html:20]: What did you mean by speed changes? I probably did not notice.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 2 above)
Comment (mpwint) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java:79]: No particular reason haha but I do think userText is a better variable name than just text.
